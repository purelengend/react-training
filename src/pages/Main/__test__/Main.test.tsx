import { renderer } from '@src/test/test-utils';

import MainPage from '..';

describe('Main Page test case', () => {
  it('should render correctly', () => {
    const mockMainPage = renderer(<MainPage />);

    expect(mockMainPage.toJSON()).toMatchSnapshot();
  });
});
