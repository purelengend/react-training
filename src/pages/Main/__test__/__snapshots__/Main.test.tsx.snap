// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Main Page test case > should render correctly 1`] = `
<main
  className="d-flex-col _main-container_6dcd6e"
>
  <div
    className="d-flex _main-content-wrapper_6dcd6e"
    id="food-list"
  >
    <div
      className="_loader_3c694b _main-loading_6dcd6e"
      id="spin"
    />
    <div
      className="d-flex-center _addition-card_df6a58"
      id="add-card"
      onClick={[Function]}
    >
      <div
        className="d-flex-col _addition-wrapper_df6a58"
      >
        <img
          alt="Plus Icon"
          className="primary-icon"
          src="/src/assets/icons/plus-icon.svg"
        />
        <p
          className="_addition-content_df6a58"
        >
          Add new dish
        </p>
      </div>
    </div>
  </div>
  <button
    className="d-flex-center _expand-btn_6dcd6e"
    data-id="0"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    SHOW MORE
  </button>
</main>
`;
